### PR TITLE
The sciencing! - EXO > SCP conversions, outfit changes, alt titles for RD (and SD!)

### PR DESCRIPTION
**Pull request by Calyxman**
## About the Pull Request

Changes 'Exo tunic' to 'Scp tunic', 'Exo cup' to 'Scp' cup, removes scientist and junior scientist's gloves, 'Exo labcoat' to 'Scp labcoat', gives the science team various themed bag storage, gives 1R, SR, scigoggles, gives RD sciHUD.
## Why It's Good For The Game

Maybe the people who do RnD should get equipment for RnD.
More flavor is always good.
Bay shit is bad.
And scientists shouldn't have the labcoat of a geneticist, they're not one.
## Changelog

:cl:
add: Added science goggles to the Researcher, Senior Researcher and a sciHUD to the Research Director!
add: Added SCP tunic to all of the Scientific Department's backpacks, so they can wear it and look nice.
add: Added 'Facility Director' as an alternate title for the Site Director.
add: Added 'Chief Science Officer', 'Head Researcher' as an alternate title for the Research Director.
del: Deleted latex gloves for Research Associate and Researcher
spellcheck: EXO assets changed to SCP assets
/:cl:

### CODE REVIEW

File: maps/site53/job/outfits.dm [hunk 1]
```diff
@@ -266,22 +266,24 @@
...
 	l_ear = /obj/item/device/radio/headset/headset_sci
-	back = /obj/item/storage/backpack/satchel
+	back = /obj/item/storage/backpack/satchel/tox
+	backpack_contents = list(/obj/item/clothing/accessory/tunic = 1)
 
 /decl/hierarchy/outfit/job/site90/crew/science/scientist
 	name = OUTFIT_JOB_NAME("Scientist")
 	uniform = /obj/item/clothing/under/scp/suittie
-	suit = /obj/item/clothing/suit/storage/toggle/labcoat/genetics
+	/obj/item/clothing/suit/storage/toggle/labcoat/science
```
Comment (Waklaidan) [inline maps/site53/job/outfits.dm]: what it is?

**Review by Waklaidan — CHANGES_REQUESTED** [review]

File: maps/site53/job/outfits.dm (hunk 1 above)
Comment (ItsTisme) [inline maps/site53/job/outfits.dm]:
```suggestion
	suit = /obj/item/clothing/suit/storage/toggle/labcoat/science
```

File: maps/site53/job/outfits.dm [hunk 2]
```diff
@@ -266,22 +266,24 @@
...
 	shoes = /obj/item/clothing/shoes/laceup
 	id_types = list(/obj/item/card/id/sciencelvl2)
-	gloves = /obj/item/clothing/gloves/latex
```
Comment (ItsTisme) [inline maps/site53/job/outfits.dm:282]: should not really matter but why remove the gloves?

**Review by ItsTisme — COMMENTED** [review]
Not really sure why swapping backpacks but im not against it. I assume you tested it localy?

**Comment by tichys** [comment]
Failing checks means no merge

File: maps/site53/job/outfits.dm (hunk 1 above)
Comment (Calyxman) [reply maps/site53/job/outfits.dm]: EXO labcoat

File: maps/site53/job/outfits.dm (hunk 2 above)
Comment (Calyxman) [reply maps/site53/job/outfits.dm:282]: They don't really have a purpose. I should probably also remove the RD and SR's gloves, actually

File: maps/site53/job/outfits.dm (hunk 1 above)
Comment (Calyxman) [reply maps/site53/job/outfits.dm]: Oops. Oversight, committing change

**Comment by Calyxman** [comment]
> Not really sure why swapping backpacks but im not against it. I assume you tested it localy?

Just to line up with the new labcoats. It's strange how Science gets the same exact bag, even the RD, and it's not even their dept colors (easy ID of a science role)

**Pull request merged**